job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1608,6 +1608,78 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+        /**
+          * <p>Configures a Weblogic deployment using the weblogic-deployer-plugin</p>
```

Comment:
Remove the <p> tags, we don't use them. And remove the XML examples from the GroovyDoc, we don't have them any more.